functions/channelCapture/channelCaptureHandlers.private.ts

```diff
@@ -0,0 +1,531 @@
+/* eslint-disable global-require */
```

Comment:
I like this approach, we can extend it to be a full multicast event system in the future if we ever need to.